plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button, Card } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './product-card.scss';
+import { Product } from '../product-list-content/product-list-content';
+
+export interface ProductCardProps {
+	type?: string;
+	product: Product;
+}
+
+function ProductCard( props: ProductCardProps ): JSX.Element {
+	const { product } = props;
+	return (
+		<Card className="woocommerce-marketplace__product-card">
+			<div className="woocommerce-marketplace__product-card__content">
+				<div className="woocommerce-marketplace__product-card__header">
+					<div className="woocommerce-marketplace__product-card__details">
+						{ product.icon && (
+							<div className="woocommerce-marketplace__product-card__icon">
+								<img
+									src={ product.icon }
+									alt={ product.title }
+								/>
+							</div>
+						) }
+						<div className="woocommerce-marketplace__product-card__meta">
+							<h2 className="woocommerce-marketplace__product-card__title">
+								{ product.title }
+							</h2>
+							<p className="woocommerce-marketplace__product-card__vendor">
+								<span>{ __( 'By ', 'woocommerce' ) }</span>
+								<a
+									href={ product.vendorUrl }
+									target="_blank"
+									rel="noopener noreferrer"
+								>
+									{ product.vendorName }
+								</a>
+							</p>
+						</div>
+					</div>
+				</div>
+				<p className="woocommerce-marketplace__product-card__description">
+					{ product.description }
+				</p>
+				<Button
+					className="woocommerce-marketplace__product-card__price"
+					href={ product.url }
+					target="_blank"
+					variant="link"
+				>
+					<span>
+						{ product.price === 0 || product.price === '0'
+							? __( 'Free download', 'woocommerce' )
+							: product.price +
+							  window.wcSettings.admin.currency.symbol }
```

Comment:
Both optional: 

- there is a helper `getAdminSetting()` function to help get settings from this global variable. 
- Is it worth adding some optional chaining in case the currency symbol isn't in the API for whatever reason?